pipeline/thumbnails_test.go

```diff
@@ -0,0 +1,79 @@
+package pipeline
+
+import (
+	"context"
+	"log"
+	"net/url"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"gopkg.in/vansante/go-ffprobe.v2"
+)
+
+func TestGenerateThumbs(t *testing.T) {
+	tests := []struct {
+		name  string
+		input string
+	}{
+		{
```

Comment:
nit: do you plan to add more test scenarios here? If not, then you may skip this `tests` part and just a have a single test.